routes/orders/order.model.js

```diff
@@ -103,9 +108,26 @@ OrderSchema.statics = {
     },
 
     getByStatus(orderStatus) {
-        return this.find({status : orderStatus})
-            .sort({createdAt: -1})
+        return this.find({ status: orderStatus })
+            .sort({ createdAt: -1 })
             .exec();
+    },
+
+    addItems(id, items) {
+        return this.findOneAndUpdate(
+            { "_id": id },
+            { $push: { items: { $each: items } } },
+            { new: true }
```

Comment:
Not sure what this `new` is doesnt seem to be [documented](https://docs.mongodb.com/manual/reference/method/db.collection.findOneAndUpdate/#definition).

Reply:
2h wasted of my life to find this... https://stackoverflow.com/questions/35626040/findoneandupdate-used-with-returnnewdocumenttrue-returns-the-original-document last answer... 
Basically, it needs that to return the updated document, not the original one...